functions/prompty/textCompletionOpenAi/openAiTextCompletion.js

```diff
@@ -1,18 +1,18 @@
-const { Configuration, OpenAIApi } = require("openai");
-
-const configuration = new Configuration({
-  apiKey: process.env.OPENAI_API_KEY,
-});
-
-module.exports.openAiTextCompletion = async function (prompt) {
-  const openai = new OpenAIApi(configuration);
-  const completion = await openai.createCompletion({
-    model: "text-davinci-003",
-    prompt: prompt,
-    max_tokens: 1024,
-    n: 3,
-    stop: null,
-    temperature: 0.7,
-  });
-  return completion.data.choices;
-};
+const { Configuration, OpenAIApi } = require("openai");
+
+const configuration = new Configuration({
+  apiKey: process.env.OPENAI_API_KEY,
+});
+
+module.exports.openAiTextCompletion = async function (prompt) {
```

Comment:
This function doesn't check the number of tries that the user has left. That should be server-side, not client-side code. This is more dangerous than the previous comment because a malicious actor could spam this endpoint to bill against our API account